packages/PriceLockup/PriceLockup.md

```diff
@@ -28,3 +40,55 @@
 ```jsx
 <PriceLockup size="medium" topText="À partir de :" price="25" signDirection="right" />
 ```
+
+### <a name="footnoteLinkWithPriceLockup">PriceLockup with FootnoteLink</a>
+
+Usage criteria:
```

Comment:
This should be `### Usage criteria` to remain consistent with other uses of the heading.